src/main/java/com/tsunderebug/speedrun4j/user/NameStyle.java

```diff
@@ -11,36 +11,18 @@
 public class NameStyle {
 
 	private String style;
-	private Map<String, String> color;
```

Comment:
Did they finally remove `color` on the backend? Hallelujah.

Reply:
They did, everyone has gradient color now.